doc/定義/機能/メモ/検索.md

```
@@ -0,0 +1,33 @@
+## 簡易検索
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)